fieldservice_google_map/views/fsm_order.xml

```diff
@@ -70,4 +70,11 @@
         </field>
     </record>
 
+    <record id="action_orders_view_map" model="ir.actions.act_window.view">
+        <field name="sequence" eval="1" />
```

Comment:
```suggestion
        <field name="sequence" eval="2" />
```
@max3903 Maybe Kanban view should be first, not the map?

Reply:
I agree 👍